front-end/src/app/tabs/dashboard/dashboard.page.html

```diff
@@ -53,7 +53,7 @@ <h1 *ngIf="!app.isInMobileMode()">{{ 'DASHBOARD.NEWS' | translate }}</h1>
             </ion-label>
             <ion-button
               color="ESNpink"
-              *ngIf="app.user.isAdministrator && !editMode"
+              *ngIf="(app.user.isAdministrator || app.user.canManageDashboard)  && !editMode"
```

Comment:
Lascerei solo agli admin, o dovremmo modificare in modo più pesante il Resource Controller delle statistiche